src/main/webapp/script.js

```diff
@@ -136,24 +136,24 @@ function updateRestaurant(result) {
     document.getElementById("restaurant-details").style.display = "flex";
     document.getElementById("no-result-text").style.display = "none";
 
-    resContainerElement.classList.remove("hide");
-    resContainerElement.scrollTop = 0;
-    document.getElementById("infowindow-restaurant-content").classList.remove("hide");
-
     updateRestaurantDetails(0);
 
-    // Show restaurant tab if hidden
-    if(resultHidden) {
-      resultHidden = false;
-      setRestaurantTab(resultHidden);
-    }
-
     // Hide next and previous buttons if only one restaurant
     if(result.length === 1) {
       document.getElementById("restaurant-buttons").style.display = "none";
     } else {
       document.getElementById("restaurant-buttons").style.display = "flex";
     }
+    resContainerElement.classList.remove("hide");
+    resContainerElement.scrollTop = 0;
```

Comment:
Can I check what is scrollTop = 0 for?

Reply:
Scrolls back to top of the element, applicable to mobile when element is larger than max height